coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -121,6 +125,17 @@ def _parse_documentation_with_symbols(self, param_identifiers,
                 param_desc = splitted[1]
                 parsed.append(self.Parameter(name=cur_param, desc=param_desc))
 
+            elif stripped_line.startswith(excp_identifiers[0]):
+                parse_mode = self.ExceptionValue
+                excp_offset = line.find(
+                    excp_identifiers[0]) + len(excp_identifiers[0])
```

Comment:
yup i guess but here we aren't using the `stripped_line`.
So the case may differ for something like:
```
    """
    This is good
        :param xyz: abc
    """
```
see the conditions for `params` im just following the same road :wink: